src/PopEngine/index.ts

```diff
@@ -220,7 +222,6 @@ export class PopEngine {
           this.setState(p, PopStateType.HIDDEN, p.opts, null, false);
 
           if (!!popOver) {
-            document.body.removeChild(popOver);
```

Comment:
Whoa removing this here in `synchronousHidePop()`? How else are we going to remove the popover from the DOM?